docs/plugins/dotnet-plugins.md

```diff
@@ -44,4 +44,16 @@ Add authentication and authorization to your .NET service using ASP.NET Core Ide
 
 :::info
 This plugin provides a robust and customizable solution for managing user authentication and authorization in your .NET applications generated by Amplication.
-:::
\ No newline at end of file
+:::
+
+### Terraform AWS Core Provisioning
```

Comment:
Let's keep the name/s as it appears in the UI (and in the Node.js plugin page as well)

![image](https://github.com/user-attachments/assets/9d5ba866-698f-42a6-97b1-01aabdac2665)